spec/draft/migration_guide.md

```diff
@@ -0,0 +1,14 @@
+# Migration Guide
+
+## Introduction
+
+The Array API Standard was based in the common APIs across multiple Python array libraries as a response to the fragmentation of the ecosystem. It was decided for the Array API Standard to be as compact as possible, meaning that it doesn't include all the creation and manipulation functions available in other libraries, but it is equally as powerful. The aim of this guide is to help users get started with the use of the Array API by providing common equivalents with the NumPy API.
+
+Depending on the version of the Array API Standard that you are intended to use, the number of total APIs may vary and this migration guide is using the latest version `v2022.12`.
```

Comment:
I think it is important to mention what version this page is using, but I also fear this can starts to be a maintenance burden. So please let me know if it is just better to refer to the "latest version" or if it is ok to flag the actual version.